library/ssl_tls.c

```diff
@@ -5702,7 +5702,7 @@ static const uint16_t ssl_preset_default_sig_algs[] = {
 
 /* NOTICE: see above */
 #if defined(MBEDTLS_SSL_PROTO_TLS1_2)
-static uint16_t ssl_tls12_preset_default_sig_algs[] = {
+static const uint16_t ssl_tls12_preset_default_sig_algs[] = {
```

Comment:
Ah, we've fixed a few of those recently but we missed some. Would you mind also doing `ssl_tls12_preset_suiteb_sig_algs` while you're at it?

Reply:
```
grep '^static.*] =' library/*.c | grep -vw const
```
shows these are the only two remaining ones except for some SHA2 tests (not important) and `ecp_curves_new.c` which I think we shouldn't touch because it'll be resynchronized with `ecp_curves.c` at some point.

Reply:
sure, done